pyproject.toml

```diff
@@ -0,0 +1,26 @@
+[tool.poetry]
+name = "ash-dal"
+version = "0.1.0"
+description = ""
+authors = ["Dmitry Marchuk <dmarchuk@remedypointsolutions.com>"]
+readme = "README.md"
+packages = [{include = "ash_dal"}]
+
+[tool.poetry.dependencies]
+python = "^3.11"
```

Comment:
not all of our projects work on 3.11, maybe we should use lower version?

Reply:
Nope :)